src/status_im/contexts/wallet/events.cljs

```diff
@@ -464,10 +464,20 @@
         :on-success on-success
         :on-error   on-error}]})))
 
+(rf/reg-event-fx
```

Comment:
since `save address` is within send perhaps we can put it in that events file? 

This one is already starting to bloat 🤰 

Or maybe we make like an events related to "contacts"/ send address - as there is all the events related to recent adderesses, contacts, my accounts too ? 🤔

Reply:
Created a new ns just for `saved-addresses`. I have a feeling this will grow.
`ns status-im.contexts.wallet.events.saved-addresses)`

Reply:
Great! 🙌🏼